src/pages/TaskList/TaskList.js

```diff
@@ -0,0 +1,79 @@
+import './taskList.scss';
+import {Link} from 'react-router-dom';
+import {Tabs, Tab} from '../../components/Tabs/index';
+import { getTasks, deleteTask, updateTask } from '../../services/tasks';
+
+export class TaskList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tasksInWeek: []
+    };
+  }
+  days = ['Sun', 'Mon', 'Thur', 'Wed', 'Th', 'Fr', 'St'];
+  date = new Date().getDay();
+
+  createTask = (day) => {
+    this.props.history.push(`tasks/newtask?day=${day}`);
+  };
+
+
+  handleDeleteTask = (id) => {
+    deleteTask(id)
+      .then(() => this.updateTaskList());
```

Comment:
you shouldn't request tasks from server again and again, you've already have it. Removed task can be deleted from existing array locally (find taks, remove element from that array tasks, make setState)